DCCpp/PacketRegister.cpp

```diff
@@ -468,6 +481,11 @@ void RegisterList::printPacket(int nReg, byte *b, int nBytes, int nRepeat) volat
   INTERFACE.print(">");
 } // RegisterList::printPacket()
 
+byte RegisterList::idlePacket[3]={0xFF,0x00,0};                           // always leave extra byte for checksum computation
```

Comment:
a minor style point on the trailing comments, it likely would be better to put the comments on the line above whatever they are documenting rather than as a trailing comment.